tests/unit/test_version.py

```diff
@@ -71,3 +83,109 @@ def test_unparsable_version(self):
 
         with self.assertRaises(ValueError):
             SaltStackVersion.parse('Drunk')
+
+
+class TestGetters(TestCase):
+
+    def test_default_module_getter(self):
+        mod = default_module_getter('socket')
+        assert mod is not None
+        assert mod.__name__ == 'socket'
+
+    def test_default_module_getter_noexist(self):
+        mod = default_module_getter('this_module_does_not_exist')
+        assert mod is None
+
+    def test_default_version_getter_version(self):
+        class mock:
+            version = '1.2.1'
+        assert default_version_getter(mock) == '1.2.1'
+
+    def test_default_version_getter___version__(self):
+        class mock:
+            __version__ = '1.2.1'
+        assert default_version_getter(mock) == '1.2.1'
+
+    def test_default_version_getter_str(self):
+        class mock:
+            __version__ = '1.2.1'
+        assert default_version_getter(mock) == '1.2.1'
```

Comment:
@dwoz Am I missing something? What's the difference between `getter_str(self)` and `getter__version__(self)` (the test before this?

Reply:
Yeah, good catch. I just removed the duplicate test.